src/Aks/Aks.Test/ScenarioTests/KubernetesTests.ps1

```diff
@@ -146,6 +146,37 @@ function Test-NewAzAks
     }
 }
 
+function Test-NewAzAksByServicePrincipal
+{
+    # Setup
+    $resourceGroupName = Get-RandomResourceGroupName
+    $kubeClusterName = Get-RandomClusterName
+    $location = "eastus"
+
+    $ServicePrincipalId = '618a2a3a-9d44-415a-b0ce-9729253a4ba9'
+    $Secret = '5E41A57vC_ODdiFb5ji-a9~C~Mp3gKt7D~'
```

Comment:
Is this true secret? Has the secret been revoked?

Reply:
The service principal has been removed.